gno.land/pkg/integration/testing_integration.go

```diff
@@ -107,12 +117,25 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 				env.Values["_logger"] = logger
 			}
 
-			// Setup "test1" default account
+			// test1 must be created outside of the loop below because it is already included in genesis so
+			// attempting to recreate results in it getting overwritten and breaking existing tests that
+			// rely on its address being static.
 			kb.CreateAccount(DefaultAccount_Name, DefaultAccount_Seed, "", "", 0, 0)
-
 			env.Setenv("USER_SEED_"+DefaultAccount_Name, DefaultAccount_Seed)
 			env.Setenv("USER_ADDR_"+DefaultAccount_Name, DefaultAccount_Address)
 
+			// Create test accounts starting from test2.
+			for i := 1; i < numTestAccounts; i++ {
```

Comment:
Why do we need to create additional test accounts?

Reply:
I would prefer this wasn't the case, it's unexpected behavior when launching the node

Reply:
What you are describing was the original intention.  See the convo with @moul above: https://github.com/gnolang/gno/pull/1471#discussion_r1432598457